tests/test_video_ironoak.py

```diff
@@ -0,0 +1,16 @@
+
+from ironoak.person import person_detector
+
+def test_video():
+
+    videoPath = "/Users/garyding/Public/github_gsoc/depthai-python/examples/models/construction_vest.mp4"
```

Comment:
Please move the video to the repository after cropping/compressing the video and check in the video. Then use relative path.

Add a list of groundtruth positions as a list to compare to and then check overlap based on IOU (you can find implementations for this).